docs/how-to-guides/installation/msi.md

```diff
@@ -0,0 +1,51 @@
+# Install standalone prompt flow CLI
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+The promptflow CLI provides commands to manage prompt flow resources.
```

Comment:
saying something like: standalone installation doesn't require a python virtual environment. User should use this when ...